Methods/Parameters/Inventory/Inventory/Program.cs

```diff
@@ -0,0 +1,314 @@
+");
+
+    var foundItem = PrintItem();
+    Console.WriteLine(@"
+    --------------------------------------------------------------------");
+    ReturnToMenu();
+
+}
+void UpdateItem()
+{
+    Console.WriteLine(@"
+    ----------------------------Update Item----------------------------
+Please enter the name of the item you want to update:");
+    var foundItem = PrintItem();
+
+Console.WriteLine(@"
+
+What would you like to update?
+1. Quantity
+2. Price
+Please select an option (1-2):");
+
+
+    int option = 0;
+    while (!int.TryParse(Console.ReadLine(), out option) || option!=1 && option!=2)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for quantity:");
+    }
+    
+    Console.WriteLine(@"
+Please enter the new value:");
+
+
+
+
+    int value = 0;
+    while (!int.TryParse(Console.ReadLine(), out value) || value > 100)
+    {
+        Console.WriteLine("Invalid input. Please enter a valid number for quantity:");
+    }
+    Console.WriteLine(@"
+--------------------------------------------------------------------");
+
+    if(option == 1)
+    {
+        foundItem.Quantity = value;
+    }
+    else
+    {
+        foundItem.Price = value;
+    }
+
+
+
+    Console.WriteLine($@"Item successfully updated.");
+
+    ReturnToMenu();
+
+    
+}
+void RemoveItem()
+{
+    Console.WriteLine(@"
+----------------------------Remove Item----------------------------
+Please enter the name of the item you want to remove:");
+
+    string findname = FindItem();
+
+    Console.WriteLine(@$"
+Are you sure you want to remove {findname} from the inventory? (Y / N) :
+
+--------------------------------------------------------------------");
+    string confirmation = Console.ReadLine().ToUpper();//needs to be a method
+    while (string.IsNullOrWhiteSpace(confirmation) || confirmation != "Y" && confirmation != "N")
+    {
+        Console.WriteLine("Invalid input. Please enter Y/N.");
+        confirmation = Console.ReadLine();
+    }
+
+    if ( confirmation == "Y")
+    {
+        items.RemoveAll(i => i.Name == findname);
+        Console.WriteLine($@"Item successfully removed.");
+    }
+    else
+    {
+        Console.WriteLine("Item was not removed.");
+    }
+
+    ReturnToMenu();
+}
+string FindItem()
+{
+    string findname ="";
+    bool itemnotfound = true;
+    while (itemnotfound)
+    {
+
+        findname = Console.ReadLine();
+        while (string.IsNullOrWhiteSpace(findname))
+        {
+            Console.WriteLine("Invalid input. Please enter a valid name.");
+            findname = Console.ReadLine().ToUpper();
+        }
+        foreach (Item i in items)
+        {
+            if (i.Name == findname)
+            {
+                itemnotfound = false;
+                
+            }
+
+
+        }
+
+        if (itemnotfound) { 
+        Console.Clear();
+        Console.WriteLine("Name not found. Try again.");
+        }
+    }
+    return findname; ;
+}
+void AddItemDisplay()
```

Comment:
No check ensures that new item names are unique when adding an item. This could lead to duplicate entries.
Ex: 
`if (items.Any(i => i.Name.Equals(name, StringComparison.OrdinalIgnoreCase)))
{
    Console.WriteLine("Item with this name already exists.");
    return;
}
`